.github/renovate.json5

```diff
@@ -0,0 +1,134 @@
+  "extends": [
+    "config:base", // https://docs.renovatebot.com/presets-config/#configbase
+    ":semanticCommits", // https://docs.renovatebot.com/presets-default/#semanticcommits
+    ":ignoreUnstable", // https://docs.renovatebot.com/presets-default/#ignoreunstable
+    "group:allNonMajor", // https://docs.renovatebot.com/presets-group/#groupallnonmajor
+    ":separateMajorReleases", // https://docs.renovatebot.com/presets-default/#separatemajorreleases
+    ":prConcurrentLimitNone", // View complete backlog as PRs. https://docs.renovatebot.com/presets-default/#prconcurrentlimitnone
+    ":prNotPending", // https://docs.renovatebot.com/presets-default/#prnotpending
+    ":prHourlyLimitNone", // https://docs.renovatebot.com/presets-default/#prhourlylimitnone
+    "docker:enableMajor", // https://docs.renovatebot.com/presets-docker/#dockerenablemajor
+  ],
+
+  // Synchronized with a 2 week sprint cycle and outside business hours.
+  // https://docs.renovatebot.com/configuration-options/#schedule
+  "schedule": ["every 2 weeks on Saturday"],
+
+  // Give ecosystem time to catch up.
+  // npm allows maintainers to unpublish a release up to 3 days later.
+  // https://docs.renovatebot.com/configuration-options/#minimumreleaseage
+  "minimumReleaseAge": "3", 
+
+  // Create PRs, but do not update them without manual action.
+  // Reduces spurious retesting in repositories that have many PRs at a time.
+  // https://docs.renovatebot.com/configuration-options/#rebasewhen
+  "rebaseWhen": "never",
+
+  // Organizational processes.
+  // https://docs.renovatebot.com/configuration-options/#dependencydashboardlabels
+  "dependencyDashboardLabels": [
+    "type: process",
+  ],
+  // https://docs.renovatebot.com/configuration-options/#prbodynotes
+  "prBodyNotes": [
+    "Reviewer is responsible for dependency update. Ensure adequate automated",
+    "or manual testing is performed before merge.",
+  ],
+
+  "packageRules": [
+    // Report impact of a dependency update via PR label.
+    {
+      "matchUpdateTypes": ["patch"],
+       "labels": ["semver: patch"],
+    },
+    {
+      "matchUpdateTypes": ["minor"],
+       "labels": ["semver: minor"],
+    },
+    {
+      "matchUpdateTypes": ["major"],
+       "labels": ["semver: major"],
+    },
+
+    // Language-specific behaviors.
+    {
+      "matchLanguages": ["go"],
+      "postUpdateOptions": ["gomodTidy"],
+    },
+
+    // Tooling & Runtime behaviors.
+    {
+      // Covers Dockerfiles, cloudbuild.yaml, and other docker-based tools.
+      "groupName": "Docker Runtimes",
+      "matchDatasources": ["docker"],
+      // TODO: Uncomment if your Dockerfiles are not included in samples.
+      // Increases build repeatability, image cache use, and supply chain security.
+      // "pinDigests": true,
+    },
+    {
+       "groupName": "GitHub Actions",
+       "matchManagers": ["github-actions"],
+       "pinDigests": true,
+    },
+
+    // Language groupings.
```

Comment:
Does this section of matching do the same things as `group:allNonMajor`?

Reply:
group:allNonMajor is a fallback for unspecific languages and things like lockfile updates (the latter which renovate recommends to handle separately from dependency updates).

This section groups nonmajor per language, so for example, all Java minor & patch updates should be grouped, but C++ would fall back into a general catch all. This way we default to a fairly aggressive grouping approach, but try to line up reviews with one language at a time to expedite the process.